lib/haml_lint/linter/README.md

```diff
@@ -308,6 +308,25 @@ Check that spaces are used for indentation instead of hard tabs.
 Option          | Description
 ----------------|-------------------------------------------------------------
 `character`     | Character to use for indentation. `space` or `tab` (default `space`)
+`width`         | Number of spaces to use for `space` indentation. (default 2)
+
+**Bad: indentation is 1 space**
+```haml
+%button
+ Hit me
+```
+
+**Bad: indentation is 4 spaces**
+```haml
+%button
+    Hit me
+```
+
+**Good: indentation is 2 spaces**
+```haml
+%button
+  Hit me
+```
```

Comment:
Do you think it would be helpful to show an example of:

```yaml
Indentation:
  character: tab
  width: 1
```

I know the Ruby style guide says spaces, but there are groups that use tabs and it might help to have that example.